libs/ngxtension/active-element/src/lib/active-element.ts

```diff
@@ -0,0 +1,20 @@
+import { DOCUMENT } from '@angular/common';
+import { assertInInjectionContext, inject, type Injector } from '@angular/core';
+import { fromEvent, map, merge, shareReplay } from 'rxjs';
+
+export function injectActiveElement(injector?: Injector) {
+	injector ?? assertInInjectionContext(injectActiveElement);
+	const doc = injector ? injector.get(DOCUMENT) : inject(DOCUMENT);
```

Comment:
suggestion: use `assertInjector` for this. You can look at the other `inject***` to see the usages.